app/forms/idv/state_id_form.rb

```diff
@@ -4,8 +4,8 @@ class StateIdForm
     include FormStateIdValidator
 
     ATTRIBUTES = %i[first_name last_name dob state_id_address1 state_id_address2
-                    state_id_city state_id_zipcode state_id_jurisdiction state_id_number
-                    same_address_as_id].freeze
+                    state_id_city state_id_zipcode state_id_jurisdiction
+                    state_id_state state_id_number same_address_as_id].freeze
```

Comment:
Just checking my understanding: `state_id_jurisdiction` corresponds to the existing field titled 'State' that is under the heading 'Address on your ID' whereas `state_id_state` corresponds to the new issuing state field?

If so, I think `state_id_state` is a bit confusing of a name. Maybe use `state_id_issuer`? IDK, that's the first thing that came to mind.